image_filters/vello_filters_cpu/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "vello_filters_cpu"
```

Comment:
I missed this in my original review, but I think this doesn't need "Vello" branding: the filters are intended to be a general solution. This might give someone the impression they are to be used together with Vello.

Reply:
That's a reasonable claim. I *think* I disagree; I think it's important to have a searchable name, and these are ultimately being developed with Vello as the "target" consumer.

But I don't feel all that strongly about it.

Reply:
If that's the case, then I guess I need to develop my own. My original discussion was entirely about this being independent of Vello.

Reply:
> I think it's important to have a searchable name

I agree. This could get a proper name of itself (in keeping with the Esperanto convention, throwing a bunch of words through a translation dictionary: "Miksi",  "Efiko", "Adapti").